stemcell_builder/stages/bosh_monit/assets/restrict-monit-api-access

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+# THESE SCRIPTS ARE EXECUTED IN LEXICAL ORDER. MAKE SURE TO CHECK IF ANYTHING ELSE (CURRENTLY BOSH FOR NATS) WRITES OTHER CGROUP BASED RULES TO AVOID INTERFERENCE. IT'S PROBABLY A GOOD IDEA TO -A DROPS AND -I ACCEPTS
```

Comment:
As mentioned above, 

> If the scripts are executed in lexical order, I'd argue that the scripts that get run should be named in the `rc` style

Reply:
changes in https://github.com/cloudfoundry/bosh-linux-stemcell-builder/pull/197